mappings/net/minecraft/obfuscate/DontObfuscate.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/obfuscate/DontObfuscate net/minecraft/obfuscate/DontObfuscate
+	COMMENT This annotation applies to some unobfuscated elements within the Minecraft
+	COMMENT source code.
+	COMMENT
+	COMMENT <p>Its behavior appears as follows:
+	COMMENT <ul>
+	COMMENT <li>The annotation itself is not obfuscated.</li>
+	COMMENT <li>If a class is annotated, it is not obfuscated.</li>
+	COMMENT <li>If a member is annotated, it and its containing class is not obfuscated,
+	COMMENT but other members in the same class may stay obfuscated.</li>
+	COMMENT </ul>
+	COMMENT
+	COMMENT <p>In addition, single-abstract-method interfaces used as lambda expressions
+	COMMENT have their single abstract method unobfuscated without this annotation.
```

Comment:
I think this is a bug in proguard more than anything

Reply:
Is it? Can't find anything at https://github.com/Guardsquare/proguard/issues

Reply:
Edited to say this is proguard's behavior than anno related